presto-hive/src/main/java/com/facebook/presto/hive/metastore/ThriftHiveMetastore.java

```diff
@@ -579,9 +590,24 @@ public void grantTablePrivileges(String databaseName, String tableName, String g
     }
 
     @Override
-    public void revokeTablePrivileges(String databaseName, String tableName, String grantee, Set<PrivilegeGrantInfo> privilegeGrantInfoSet)
+    public void revokeTablePrivileges(String databaseName, String tableName, String grantee, Set<PrivilegeGrantInfo> requestedPrivileges)
     {
+        checkArgument((requestedPrivileges.stream()
```

Comment:
This is good but I suggest that you extract the method:

```
    private boolean containsAllPrivilege(Set<PrivilegeGrantInfo> requestedPrivileges)
    {
        return requestedPrivileges.stream()
                .anyMatch(privilege -> privilege.getPrivilege().equalsIgnoreCase("all"));
    }
```

And put line like this
```
        checkArgument(!containsAllPrivilege(requestedPrivileges), "ALL not supported in PrivilegeGrantInfo");
```
here.

Also put same line in grantTablePrivileges as it does not behave correctly for `ALL` neither.